runtime/src/iree/hal/local/loaders/vmvx_module_loader.c

```diff
@@ -441,27 +457,33 @@ typedef struct iree_hal_vmvx_module_loader_t {
   iree_allocator_t host_allocator;
   iree_vm_instance_t* instance;
   iree_vm_module_t* vmvx_module;
+  iree_host_size_t module_count;
+  iree_vm_module_t* modules[];
```

Comment:
Good point! Renamed the arguments to user_modules and then removed the standalone vmvx_module so that the list is all common_modules across loaded executables.